packages/android_intent/android/src/test/java/io/flutter/plugins/androidintent/MethodCallHandlerImplTest.java

```diff
@@ -283,4 +285,38 @@ public void onMethodCall_setsDataAndType() {
     assertEquals(type, intent.getType());
     assertEquals(data, intent.getData());
   }
+
+  @Test
+  public void onMethodCall_setsIgnoredPackages() {
+    sender.setApplicationContext(context);
+    Map<String, Object> args = new HashMap<>();
+    args.put("action", "foo");
+    ArrayList<String> ignoredPackages = new ArrayList<String>();
+    ignoredPackages.add("io.flutter.plugins.androidintent");
+    args.put("ignoredPackages", ignoredPackages);
+    Result result = mock(Result.class);
+
+    methodCallHandler.onMethodCall(new MethodCall("launch", args), result);
+
+    verify(result, times(1)).success(null);
+    Intent intent = shadowOf((Application) context).getNextStartedActivity();
+    assertNotNull(intent);
+    assertEquals("foo", intent.getAction());
+    assertNotEquals("io.flutter.plugins.androidintent", intent.getPackage());
+  }
+
+  @Test
+  public void onMethodCall_showChooser() {
```

Comment:
I'm not sure what this test is verifying, either. Shouldn't there be some verification that the new "show chooser" behavior did something? Maybe I'm overlooking some details in this test, but it looks like it's verifying the same kind of behavior that we'd expect if we ran the regular `Intent` behavior, right?

Reply:
You're right, I will fix this test.